api/v1beta1/bgppeer_webhook_test.go

```diff
@@ -45,6 +45,22 @@ func TestValidateBGPPeer(t *testing.T) {
 			},
 			expectedError: "Invalid RouterID",
 		},
+		{
+			desc: "BGPPeer with different RouterID",
```

Comment:
Do we have a test with non-zero routerids set to all the peers (in unit or e2e,  to make sure the happy path works)